assets/chat/css/generify.scss

```diff
@@ -62,6 +62,57 @@
     }
 }
 
+@keyframes peek {
+    0% {
+        transform: translate(-110%, 0);
+    }
+    
+    50% {
+        transform: translate(-33%, 0);
+    }
+
+    100% {
+        transform: translate(-110%, 0);
+    }
+}
+@keyframes reverse-peek {
+    0% {
+        transform: translate(110%, 0);
+    }
+    
+    50% {
+        transform: translate(33%, 0);
+    }
+
+    100% {
+        transform: translate(110%, 0);
+    }
+}
+
+.generify-peek {
+    clip-path: polygon(0 -100%, 100% -100%, 100% 200%, 0 200%);
+
+    & > .generify-container {
+        animation: peek 6s ease-in-out 4;
+
+        @at-root .pref-animateforever &, &:hover {
+            animation-iteration-count: infinite;
+        }
+
+        @at-root .reverse & {
+            animation: reverse-peek 6s ease-in-out infinite !important;
```

Comment:
animation shouldn't play infinitely when reversed it's animateforever/hover. also why is this `!important`? and i think you just want to change `animation-name` for this

Reply:
animation change does work, and doesn't require an `!important` - I've changed this now.